### PR TITLE
Add VSCode dev container to build and serve hugo page

### PR DESCRIPTION
**Pull request by lucas-koehler**
* Add VSCode dev container config to this repo enabling to open, build, and serve this page inside a container.
* Add VSCode tasks for building and serving.
* Adapt README

### CODE REVIEW

**Review by eneufeld — COMMENTED** [review]
Didn't test, but I like the idea.
Could you add `ms-vscode-remote.remote-containers` to the list of recommended plugins and thus a .vscode folder?

**Comment by lucas-koehler** [comment]
@eneufeld VSCode automatically suggests the extension when the workspace is not launched as a dev container. Thus, I would not explicitly add it as it is then also recommended inside the dev container. A vscode folder is present for the tasks. What do you think?

**Comment by jfaltermeier** [comment]
I've tested it and I was asked to reopen the folder in a container, which worked fine. 
However when I run `hugo server` or use the `Serve Drafts` task I just get an empty page at `http://localhost:1313/emfcloud/`
Am I missing something?

**Comment by lucas-koehler** [comment]
@jfaltermeier Good find! Apparently the page requires hugo extended and only the normal version was configured. For me it worked because I had built the page earlier. Now it should work (you need to rebuild the devcontainer though. VScode should prompt you for that).

File: README.md [hunk 1]
```diff
@@ -9,20 +9,22 @@ Please check the [Syna documentation](https://about.okkur.org/syna/docs/). The S
 
 ## Quick start
 
-### Prerequisites
-In case you use GitPod (see button above) to contribute to the website, you can skip the prerequisites section.
+### Open in VSCode dev container
 
-  * Get submodules `git submodule init && git submodule update`
```
Comment (jfaltermeier) [inline README.md:15]: Do we still need the submodules?

**Review by jfaltermeier — COMMENTED** [review]
VSCode did ask me to rebuild, so this also works fine. 
At first the page was still blank. 
I then saw the `.gitmodules` file and fetched the submodules
It worked on the next try. So either this was a caching issue or the submodules are still needed. The submodule part was removed from the Readme.

File: README.md (hunk 1 above)
Comment (lucas-koehler) [reply README.md:15]: Yes, we do still need it. Good catch! It was the same issue that I began with an existing checkout of the repository. I'll retest with a fresh clone.

**Comment by lucas-koehler** [comment]
@jfaltermeier I readded the submodule entry and tested in a freshly cloned repository. For me it worked now.

**Comment by eneufeld** [comment]
> @eneufeld VSCode automatically suggests the extension when the workspace is not launched as a dev container. Thus, I would not explicitly add it as it is then also recommended inside the dev container. A vscode folder is present for the tasks. What do you think?

ok, if the plugin is suggested automatically fine by me. Are there any other special plugins needed or recommended? if so maybe just bundle all of them?

**Review by jfaltermeier — APPROVED** [review]

**Comment by lucas-koehler** [comment]
> > @eneufeld VSCode automatically suggests the extension when the workspace is not launched as a dev container. Thus, I would not explicitly add it as it is then also recommended inside the dev container. A vscode folder is present for the tasks. What do you think?
> 
> ok, if the plugin is suggested automatically fine by me. Are there any other special plugins needed or recommended? if so maybe just bundle all of them?

There are two recommended ones and no required one but they are automatically installed in the dev container. So using the dev container, no plugins (except for the remote one) need to be installed :) Thus, I would not recommend them additionally.

**Pull request merged**